[2025-01-10T17:10:04.838+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:10:04.852+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:09:00+00:00 [queued]>
[2025-01-10T17:10:04.860+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:09:00+00:00 [queued]>
[2025-01-10T17:10:04.862+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:10:04.876+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 17:09:00+00:00
[2025-01-10T17:10:04.884+0000] {standard_task_runner.py:72} INFO - Started process 1102 to run task
[2025-01-10T17:10:04.887+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T17:09:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmphbyfqvf4']
[2025-01-10T17:10:04.891+0000] {standard_task_runner.py:105} INFO - Job 133: Subtask append_to_database
[2025-01-10T17:10:04.943+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T17:09:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T17:10:05.030+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:09:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T17:09:00+00:00'
[2025-01-10T17:10:05.033+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:10:20.885+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T17:10:20.887+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T17:10:20.895+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:10:20.897+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T17:09:00+00:00, execution_date=20250110T170900, start_date=20250110T171004, end_date=20250110T171020
[2025-01-10T17:10:20.937+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T17:10:20.953+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:10:20.958+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
